tools/IceRpc.Slice.Tools/IceRpc.Slice.Tools.props

```diff
@@ -49,6 +49,7 @@
   <ItemDefinitionGroup>
     <SliceFile>
       <OutputDir>$(MSBuildProjectDirectory)/generated</OutputDir>
+      <Rpc>icerpc</Rpc>
```

Comment:
This ItemDefinitionsGroup is where we specify the default values?

Reply:
Yes these are the defaults for all SliceFile items